src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -88,6 +87,14 @@ public void traverse() {
         logger.info(countedFilter.getSummaryLine());
     }
 
+    /**
+     * Returns an interval for the read.
+     * Note: some walkers must be able to work on any read, including those whose coordinates do not form a valid SimpleInterval.
+     * So here we check this condition and create empty intervals for those bad reads.
```

Comment:
I think this should say `create null intervals` empty interval sounds like we have a special type of empty interval.